packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1446,12 +1446,25 @@ export const components: ThemeOptions["components"] = {
   },
   MuiSvgIcon: {
     defaultProps: {
-      fontSize: "inherit",
       color: "inherit",
+      fontSize: "inherit",
     },
     styleOverrides: {
       root: ({ theme }) => ({
         fontSize: `${theme.typography.ui.lineHeight}rem`,
+
+        "&.Icon-sizeSmall": {
```

Comment:
@josesolano-okta -- thoughts for the best token use here?

We'll have tokens for these in 2.0. Until then, should we point to a font size directly?